qchem/pennylane_qchem/qchem.py

```diff
@@ -690,6 +690,94 @@ def hf_state(n_electrons, m_spin_orbitals):
     return np.array(hf_state_on)
 
 
+def ph_pphh_wires(ph_confs, pphh_confs, wires=None):
```

Comment:
@agran2018 I have to admit, I find both the function and the argument names a bit obfuscated 🤔

Reply:
Yes, I agree. What about this: ``convert_ph_to_wires(ph, pphh, wires=None)`` ?

Reply:
Or ``particle_hole_wires``? Or ``excitations_to_wires``?